save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/TestSuitesSourceController.kt

```diff
@@ -328,6 +314,41 @@ class TestSuitesSourceController(
         }
         .map { it.toDto() }
 
+    /**
+     * @param executionId
+     * @return selected [TestSuitesSourceDto] with versions for [com.saveourtool.save.entities.Execution] found by provided values
+     */
+    @PostMapping("/download-snapshot-by-execution-id", produces = [MediaType.APPLICATION_OCTET_STREAM_VALUE])
```

Comment:
Btw, we decide to use swagger annotations, but it clogs up so much, as for me 👎

Reply:
+1

Reply:
lines 317-336 - blablabla
lines 337-350 - code